plugins/NebulaTextures/src/gui/NebulaTexturesDialog.hpp

```diff
@@ -0,0 +1,82 @@
+/*
+ * Nebula Textures plug-in for Stellarium
+ *
+ * Copyright (C) 2024 ultrapre@github.com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef NEBULATEXTURESDIALOG_HPP
+#define NEBULATEXTURESDIALOG_HPP
+
+#include "StelDialog.hpp"
+#include <QString>
+
+class Ui_nebulaTexturesDialog;
```

Comment:
Please change case for name of related file or change the name of class - please check similar parts in other plugins

Reply:
Sorry but I can't understand what's the problem

Reply:
ok it's done